tensornetwork/tn_keras/dense.py

```diff
@@ -71,22 +71,24 @@ def build(self, input_shape: List[int]) -> None:
       raise ValueError('The last dimension of the inputs to `Dense` '
                        'should be defined. Found `None`.')
 
-    self.a_var = self.add_weight(name='a',
-                                 shape=(input_shape[-1], self.decomp_size),
-                                 trainable=True,
-                                 initializer=self.kernel_initializer)
-    self.b_var = self.add_weight(name='b',
-                                 shape=(self.decomp_size, self.output_dim),
-                                 trainable=True,
-                                 initializer=self.kernel_initializer)
+    self.a_var = self.add_weight(
+        name='a',
+        shape=(input_shape[-1], self.decomp_size),
+        trainable=True,
+        initializer=self.kernel_initializer)
+    self.b_var = self.add_weight(
+        name='b',
+        shape=(self.decomp_size, self.output_dim),
+        trainable=True,
+        initializer=self.kernel_initializer)
     self.bias_var = self.add_weight(
         name='bias',
         shape=(self.output_dim,),
         trainable=True,
         initializer=self.bias_initializer) if self.use_bias else None
     super(DenseDecomp, self).build(input_shape)
 
-  def call(self, inputs: tf.Tensor) -> tf.Tensor:
+  def call(self, inputs: tf.Tensor, **kwargs) -> tf.Tensor:  # pylint: disable=unused-argument
```

Comment:
Why is this unused?

Reply:
I added **kwargs because tensorflow-keras has it as well.

Reply:
And its also unsed there